### PR TITLE
Clarify language around exclusions

### PR DESCRIPTION
**Pull request by elharo**
@michael-o

### CODE REVIEW

File: content/apt/pom.apt.vm [hunk 1]
```diff
@@ -567,10 +567,13 @@ Display parameters as parsed by Maven (in canonical form and as a list of tokens
 
 *** {Exclusions}
 
-  Exclusions tell Maven not to include the
-  specified project that is a dependency of this dependency (in other words, its transitive
-  dependency). For example, the <<<maven-embedder>>> requires <<<maven-core>>>, and we do not
-  wish to use it or its dependencies, then we would add it as an <<<exclusion>>>.
+  It is sometimes useful to limit  a dependency's transitive dependencies. A dependency may have incorrectly
```
Comment (michael-o) [inline content/apt/pom.apt.vm:570]: double space

File: content/apt/pom.apt.vm [hunk 2]
```diff
@@ -580,7 +583,7 @@ Display parameters as parsed by Maven (in canonical form and as a list of tokens
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-embedder</artifactId>
-      <version>2.0</version>
+      <version>3.8.6</version>
```
Comment (michael-o) [inline content/apt/pom.apt.vm:586]: I think we even have a property for this in the POM.

**Review by michael-o — CHANGES_REQUESTED** [review]

File: content/apt/pom.apt.vm (hunk 2 above)
Comment (elharo) [reply content/apt/pom.apt.vm:586]: which property is that?

File: content/apt/pom.apt.vm (hunk 1 above)
Comment (elharo) [reply content/apt/pom.apt.vm]: fixed

File: content/apt/pom.apt.vm (hunk 2 above)
Comment (michael-o) [reply content/apt/pom.apt.vm:586]: currentStableVersion

Comment (elharo) [reply content/apt/pom.apt.vm]: done

**Review by michael-o — APPROVED** [review]

**Pull request merged**